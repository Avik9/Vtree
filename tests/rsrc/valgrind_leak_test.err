==5622== Memcheck, a memory error detector
==5622== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==5622== Using Valgrind-3.13.0 and LibVEX; rerun with -h for copyright info
==5622== Command: bin/vtree -q tests/rsrc/test_tree
==5622== 
==5622== 
==5622== HEAP SUMMARY:
==5622==     in use at exit: 0 bytes in 0 blocks
==5622==   total heap usage: 12 allocs, 12 frees, 202,472 bytes allocated
==5622== 
==5622== All heap blocks were freed -- no leaks are possible
==5622== 
==5622== For counts of detected and suppressed errors, rerun with: -v
==5622== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
