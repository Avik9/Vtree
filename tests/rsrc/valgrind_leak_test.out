 tests/rsrc/test_tree
   S
      d
      a
      c
      b
